pages/index.md

```diff
@@ -1,20 +1,77 @@
-title: Plasma Hack Week: June 28 – July 2, 2021
+title: Plasma Hack Week
 hidetitle: True
 
+<!-- Feature Cards -->
+<div class="feature-row" style="margin-bottom: 16px">
+    <!-- Feature 1 -->
+    <div class="feature-column">
+        <a class="feature-link" href="https://discord.gg/HdsZkp9M35">
+        <div class="feature-card" 
+              style="background: linear-gradient(to left, 
+                                 var(--plasmapy-darkblue) 0%,
+                                 var(--plasmapy-light-red) 200%);">
+            <div>
+                <h1 style="color: #d8d8d8; font-weight: bold">
+                    Discord
+                </h1>
+            </div>
+        </div>
+        </a>
+    </div>
+    <!-- Feature 2 -->
+    <div class="feature-column">
+        <a class="feature-link" href="2021/about">
+        <div class="feature-card" 
+              style="background-image: none;
+                     background-color: var(--plasmapy-darkblue)">
+            <div>
+                <h1 style="color: #d8d8d8; margin-bottom: 18px">
+                    2021 Hack Week
+                </h1>
+                <img src="/images/hack_logo(v7).png" alt="" style="max-width: 90%">
+            </div>
+        </div>
+        </a>
+    </div>
+    <!-- Feature 3 -->
+    <div class="feature-column">
+        <a class="feature-link" href="2021/registration">
+        <div class="feature-card" 
+              style="background: linear-gradient(to right, 
+                                 var(--plasmapy-darkblue) 0%,
+                                 var(--plasmapy-bluegreen) 200%);">
+            <div>
+                <h1 style="color: #d8d8d8; font-weight: bold">
+                    Registration
+                </h1>
+            </div>
+        </div>
+        </a>
+    </div>
+</div>
+
 Hello world!  We are pleased to announce that the inaugural **Plasma
 Hack Week** will be held remotely from June 28 – July 2, 2021.  In the
 week before the event, we will hold prequel tutorials on topics such
 as Python, git & GitHub, contributing to an open source project, and
 writing clean scientific software.  
 
-## What is a hack week?
+## What is a hack week and hacking?
+
+The term hacking has a broad range of use in modern day slang.  However,
+it is often implied that hacking is done by some prodigy or expert.
+This is not necessarily the case for the **Plasma Hack Week**.  Hacking
```

Comment:
```suggestion
This is not the case for the **Plasma Hack Week** or scientific coding in general.  Hacking
```

Reply:
I tweaked your suggestion a little bit, so let me know what you think.